### ISSUE TITLE
Fehlermeldung verschwindet nicht

### ISSUE THREAD

**Issue by Mucki526** [jlawyerorg/j-lawyer-org#1787]
Wenn ich j-lawyer client auf dem Laptop (Windows 10 / Windows 11 laufen habe und den Laptop zuklappe geht er in den Ruhemodus. Beim Aufwecken des Rechners kommt dann von J-lawyer "Connection lost". Problem - die Meldung geht nicht weg und lässt sich auch nicht löschen, selbst wenn die Verbindung längst wieder steht. Ich hier hilft nur j-lawyer über den Taskmanager abzuschießen und neu zu starten.  

**Describe the bug / Fehlerbeschreibung**
(todo: a description of the bug / eine Beschreibung des Fehlers)

**To Reproduce / Schritte zum Nachvollziehen**

1. Go to / Gehe zu '...'
2. Klicke auf / Click on '....'
4. Error message / Fehlermeldung: '...'

**Expected behavior / Erwartetes Verhalten**
Eine Beschreibung des erwarteten Verhaltens.

**Screenshots**
Falls relevant, Screenshot beifügen.

**Environment / verwendete Umgebung:**
 - OS / Betriebssystem:
 - j-lawyer.org Version:

**Additional context / Sonstiges**

**Comment by j-dimension** [comment]
@Mucki526  welche Verbindungsart wird verwendet? (im Verbindungsprofil)

**Comment by Mucki526** [comment]
Die Fritzboxen der Kanzlei und zuhause sind im VPN verbunden. Ich spreche daheim direkt den Server der Kanzlei über seine IP an.

**Comment by Mucki526** [comment]
Standard

**Comment by j-dimension** [comment]
@Mucki526 tritt dieses Problem noch auf?

**Comment by j-dimension** [comment]
Ist das noch ein Problem?

**Comment by Mucki526** [comment]
Ja, es ist nach wie vor ein Problem. Wir haben nach dem Ruhemodus, wenn j-lawyer am Laptop offen war beim Deckel zuklappen regelmäßig noch nicht löschbare Fehlermeldungen des Clients. Dann hilft es nur über den Taskmanager den Task von j-lawyer zu beenden.
